pixorama_server/deploy/aws/terraform/config.auto.tfvars

```diff
@@ -25,31 +28,37 @@ enable_redis = false
 # *.example.com to your certificate. You will need to create one certificate
 # for your main region and one for use with Cloudfront that resides in the
 # us-east-1 region.
```

Comment:
There is a reason these were filled in. Without them it will be impossible to host this on AWS.